test/integration/dockerized_job_conf.yml

```
@@ -10,12 +10,14 @@ execution:
     local_docker:
       runner: local
       docker_enabled: true
+      # require_container: true
```

Are the comments in this and the following file needed?

---

I remember that I played around with them. Lets just try to enable them and see what fails. I think this makes the tests stronger since it ensures that the tests really use containers and do not fall back to conda (which is the default).

```suggestion
      require_container: true
```